app/main.py

```diff
@@ -2,0 +1,34 @@
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 heals: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = heals
+        self.hidden = hidden
```

Comment:
Code Style: You should use descriptive and correct variable names. In this case, instead of `heals`, you should use `health`. Refer to point 4 in the checklist.